util/tera.py

```diff
@@ -1,23 +1,90 @@
+import requests
+import os
+import pyzipper
+
 from selenium.webdriver.common.by import By
-from time import sleep
 import common.constant as Constant
+from util.profile import ChromeProfile
+
+
+class TeraBox:
+
+    def __init__(self, path, password):
+        email = Constant.env["EMAIL"]
+        email = email.split(':')
+        profile = ChromeProfile(email[0], email[1], email[2])
+        self.driver = profile.retrieve_driver()
+        self.cookie = None
+        self.path = path
+        self.password = password
+        profile.start()
+
+    def login(self):
```

Comment:
Should login be inside of class init since this class always need to login terabox to be able to use?